### PR TITLE
Enable test that was never enabled

### LINKED ISSUE #15711: AroundConstructTestServlet.testInterceptorNotCalledTwice does not pass
https://github.com/OpenLiberty/open-liberty/blob/1d160bafc1969d4bd2a5a87f763b20a220704664/dev/com.ibm.ws.cdi.ejb_fat/test-applications/aroundConstructApp.war/src/com/ibm/ws/cdi12/test/aroundconstruct/AroundConstructTestServlet.java#L143

`testInterceptorNotCalledTwice` was previously not properly marked as a `@Test` and so has not been run in a while. Don't know how long. When I re-enabled it, it fails, the constructor interceptor appearing to be run twice.

### PR DESCRIPTION
**Pull request by benjamin-confino**
fixes #15711

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Review by Azquelt — APPROVED** [review]

File: dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi/ejb/apps/aroundconstruct/interceptors/ConstructInterceptor.java [hunk 1]
```diff
@@ -31,16 +29,9 @@
     @Inject
     AroundConstructLogger logger;
 
-    @Inject
-    StatelessAroundConstructLogger statelessLogger;
-
     @AroundConstruct
     public Object intercept(InvocationContext context) throws Exception {
-        //If the stateless bean is being intercepted set this in the stateless logger
         Class<?> declaringClass = context.getConstructor().getDeclaringClass();
```
Comment (Azquelt) [inline dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi/ejb/apps/aroundconstruct/interceptors/ConstructInterceptor.java:34]: This line can go as well now.

**Review by Azquelt — APPROVED** [review]

**Review by Azquelt — CHANGES_REQUESTED** [review]
Changes look good but need to reenable the disabled test.

**Review by Azquelt — APPROVED** [review]

**Comment by benjamin-confino** [comment]
#run-libby-bot

**Pull request merged**